### PR TITLE
[Language center view]: Make language center view visibility and backend functionality configurable

### PR DESCRIPTION
**Pull request by rundtjan**
By using the LANGUAGE_CENTER_VIEW_ENABLED env variable, (applicable both on frontend and backend), one can configure whether this functionality is available or not. If the env variable is not set, the default value is "true" (as in Toska's environment).

Also hiding the "Cause frontend crash"-button in FD environment and enabling configuration of concurrent worker amount in backend.

### CODE REVIEW

File: services/backend/src/config/index.js [hunk 1]
```diff
@@ -47,6 +47,11 @@ const { DB_URL_KONE, DB_URL_USER, SECRET_TOKEN, SIS_DB_URL, SIS_UPDATER_URL, CRY
 
 const rootOrgId = process.env.ROOT_ORG_ID || 'hy-university-root-id'
 
+const languageCenterViewEnabled =
+  process.env.LANGUAGE_CENTER_VIEW_ENABLED == null ? true : process.env.LANGUAGE_CENTER_VIEW_ENABLED === 'true'
```
Comment (mattirasanen) [inline services/backend/src/config/index.js:51]: Would it make also like this?
`const languageCenterViewEnabled = !process.env.LANGUAGE_CENTER_VIEW_ENABLED
  ? true
  : process.env.LANGUAGE_CENTER_VIEW_ENABLED === 'true'`

File: services/frontend/src/conf.js [hunk 2]
```diff
@@ -29,3 +29,9 @@ export const builtAt = process.env.REACT_APP_BUILT_AT || ''
 export const serviceProvider = process.env.REACT_APP_SERVICE_PROVIDER
   ? process.env.REACT_APP_SERVICE_PROVIDER.toLowerCase()
   : 'toska'
+
+// Variable that can be used to disable the language center view, by default the view is enabled
+export const languageCenterViewEnabled =
+  process.env.REACT_APP_LANGUAGE_CENTER_VIEW_ENABLED == null
```
Comment (mattirasanen) [inline services/frontend/src/conf.js:35]: Should this work also with?
`const languageCenterViewEnabled = !process.env.LANGUAGE_CENTER_VIEW_ENABLED
  ? true
  : process.env.LANGUAGE_CENTER_VIEW_ENABLED === 'true'`

File: services/backend/src/config/index.js (hunk 1 above)
Comment (rundtjan) [reply services/backend/src/config/index.js:51]: This solution would give the wrong result, as process.env.LANGUAGE_nönönö is a string, so if process.env.LANGUAGE_nönönö is set, this (!process.env.LANGUAGE_nönönö) will be evaluated as false, no matter what string that is passed to it.

The evaluation with process.env.LANGUAGE_nönönö == null catches cases where the variable is undefined.

File: services/frontend/src/conf.js (hunk 2 above)
Comment (rundtjan) [reply services/frontend/src/conf.js:35]: Same as :arrow_up:

**Comment by mattirasanen** [comment]
FD approved

**Review by valtterikantanen — APPROVED** [review]

**Pull request merged**